sections/summary.tex

```diff
@@ -0,0 +1,108 @@
+This gives the intuition of how similar authors are based on the topics they write, and categories in the topics they cover.
+It is seen that it is difficult to find specifically why authors are similar, because they often write about many different subjects.
+From the category similarities there are obvious high similarity pairs, such as 'Sport-avis' and 'Morsø Sport' where they both include sports articles.
+
+For the taxonomy-topic model, it was found that the topics in general were the most understandable of all topic model, which made sense with it having the highest topic coherence.
+The taxonomy-topic model also was able to separate words without meaning into their own topics, which allows the model to apply an extra layer of preprocessing, automatically filtering away irrelevant words into topics
+Finally it was described how the layered structure of \gls{pam} gives further opportunities for ways to recommend articles.
+
+The analysis shows that each model has its own usefulness in supporting recommendation of articles depending on what kind of recommendation is wanted.
+
+%appendix
+As further experimentation and exploration, we tested what would happen if we did further preprocessing, if we included multiple topic distributions in our models, and if we used the author and category metadata in the \gls{pam}.
+
+For further preprocessing we tested what effect including stemming in our dataset would have on the models.
+With stemming on the \gls{lda} model, it was seen that there were a lot fewer unique words.
+When looking at the topics of the model, it was seen that they generally did not change much, but may be slightly more understandable with fewer words with the same meaning.
+
+One of the new models we made is called the author-category model.
+This model includes both an author-topic distribution and a category-topic distribution, and uses both to draw word topics.
+This model has only a slightly higher topic coherence of 0.390 compared to the author-topic and category-topic model, and the topics are still not very understandable.
+Though, while this is the case, having two topic distributions will give more possibilities for recommendation of articles compared to having one topic distribution.
+
+We also made two other models called the Author-Doc and Category-Doc model.
+These models are made by combining the \gls{lda} model with the author-topic and category-topic mode, and also uses both distributions to draw word topics.
+We run these models using the same hyperparameters as all the other models and they get similar results to the standard \gls{lda} model.
+Author-doc gets a topic coherence of 0.543 and category-doc gets 0.530.
+
+Finally, we tested what would happen if we ran \gls{pam} with just the author and category metadata.
+A four-layer \gls{pam} was used for both of these models, but otherwise the same structure is used as with the taxonomy-topic model.
```

Comment:
```suggestion
A four-layer \gls{pam} was used for both of these models, but otherwise, the same structure is used as with the taxonomy-topic model.
```